Index: libtool-2.2.2/libltdl/config/ltmain.m4sh
===================================================================
--- libtool-2.2.2.orig/libltdl/config/ltmain.m4sh	2008-04-06 12:14:22.000000000 +0000
+++ libtool-2.2.2/libltdl/config/ltmain.m4sh	2008-04-06 12:18:40.000000000 +0000
@@ -5249,6 +5249,9 @@
 	    revision="$number_minor"
 	    lt_irix_increment=no
 	    ;;
+	  *)
+	    func_fatal_configuration "$modename: unknown library version type \`$version_type'"
+	    ;;
 	  esac
 	  ;;
 	no)
